### PR TITLE
[PT] Translation of Ecto/Querying

### PR DESCRIPTION
**Pull request by pedrobarbiero**

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @pedrobarbiero! I'll give the @elixirschool/portuguese translators some time to review this 👍

File: pt/lessons/ecto/querying.md [hunk 1]
```diff
@@ -0,0 +1,443 @@
+---
+version: 1.0.0
+title: Querying
+---
+
+{% include toc.html %}
+
+Nesta lição, estaremos construindo o aplicativo `Example` e o catálogo de filmes que configuramos na [lição anterior](./associations)
```
Comment (thiamsantos) [inline pt/lessons/ecto/querying.md]:
```suggestion
Nesta lição, estaremos construindo o aplicativo `Example` e o catálogo de filmes que configuramos na [lição anterior](./associations).
```

Missing period.

Comment (thiamsantos) [inline pt/lessons/ecto/querying.md]:
```suggestion
Nesta lição, estaremos construindo a aplicação `Example` e o catálogo de filmes que configuramos na [lição anterior](./associations)
```
WDYT of `aplicação`?

File: pt/lessons/ecto/querying.md [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+
+## Buscando Registros com `Ecto.Repo`
+
+Lembre-se de que um repositório no Ecto é mapeado para um armazenamento de dados, como nosso banco de dados Postgres.
+Toda a comunicação com o banco de dados será feita usando este repositório.
+
+Podemos realizar simples consultas diretamente em nosso `Example.Repo` com a ajuda de uma porção de funções.
```
Comment (thiamsantos) [inline pt/lessons/ecto/querying.md]:
```suggestion
Podemos realizar simples consultas diretamente em nosso `Example.Repo` com a ajuda de uma algumas de funções.
```
`algumas` seems to be a more clear translation of handful?

File: pt/lessons/ecto/querying.md [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+
+### Buscando Registros por ID
+
+Nós podemos usar a função `Repo.get/3` para carregar um registro vindo do banco de dados pelo seu ID. Essa função requer dois argumentos: uma estrutura "queryable" e o ID do registro para recuperar do banco de dados. Ela retorna uma estrutura descrevendo o registro encontrado, caso exista. Retorna `nil` se nenhum registro for encontrado.
```
Comment (thiamsantos) [inline pt/lessons/ecto/querying.md]:
```suggestion
Nós podemos usar a função `Repo.get/3` para carregar um registro vindo do banco de dados pelo seu ID. Essa função requer dois argumentos: uma estrutura "queryable" e o ID do registro para recuperar do banco de dados. Ela retorna uma struct descrevendo o registro encontrado, caso exista. Retorna `nil` se nenhum registro for encontrado.
```
WDYT about keeping `struct` here?

File: pt/lessons/ecto/querying.md (hunk 1 above)
Comment (pedrobarbiero) [reply pt/lessons/ecto/querying.md]: `aplicação` really makes more sense.

File: pt/lessons/ecto/querying.md (hunk 3 above)
Comment (pedrobarbiero) [reply pt/lessons/ecto/querying.md]: I was in doubt when use `struct` or `estrutura`

**Comment by pedrobarbiero** [comment]
Thank you @thiamsantos for review. 
It's ready to re-check.

**Comment by anderkonzen** [comment]
Thanks @pedrobarbiero for the awesome work, and @angelo-moreira and @thiamsantos for the review. I'll go ahead and merge the changes. 💯

**Pull request merged**